SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -203,16 +206,26 @@ - (void)didEnterStateStarted {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
     if (self.configuration.lifecycleConfig.tcpDebugMode) {
+        // secondary transport manager is not used
+        self.secondaryTransportManager = nil;
         self.proxy = [SDLProxy tcpProxyWithListener:self.notificationDispatcher
                                        tcpIPAddress:self.configuration.lifecycleConfig.tcpDebugIPAddress
-                                            tcpPort:@(self.configuration.lifecycleConfig.tcpDebugPort).stringValue];
+                                            tcpPort:@(self.configuration.lifecycleConfig.tcpDebugPort).stringValue
+                          secondaryTransportManager:self.secondaryTransportManager];
     } else {
-        self.proxy = [SDLProxy iapProxyWithListener:self.notificationDispatcher];
+        self.secondaryTransportManager = [[SDLSecondaryTransportManager alloc] initWithStreamingProtocolListener:self];
+        self.proxy = [SDLProxy iapProxyWithListener:self.notificationDispatcher
+                          secondaryTransportManager:self.secondaryTransportManager];
     }
 #pragma clang diagnostic pop
 
-    if (self.streamManager != nil) {
-        [self.streamManager startWithProtocol:self.proxy.protocol];
+    // if secondary transport manager is used, streaming media manager will be started through
+    // onAudioServiceProtocolUpdated and onVideoServiceProtocolUpdated
+    if (self.secondaryTransportManager == nil) {
+        if (self.streamManager != nil) {
```

Comment:
Can this be streamlined into the above `if`?

Reply:
Sure, done in commit da0b0fcf5b240b479d4360de430ee5ac9681e9d6.